Parser/Device/DeviceParserAbstract.php

```diff
@@ -483,6 +488,7 @@ abstract class DeviceParserAbstract extends ParserAbstract
         'T3' => 'Trevi',
         'TU' => 'Tunisie Telecom',
         'TR' => 'Turbo-X',
+        'T10' => 'True',
```

Comment:
In order to keep that compatible with Matomo we need to use 2 letter short codes for now. Hope we will be able to change that with Matomo 4 and maybe even remove the short codes here completely...

Reply:
The closest competitor only count 4757 positions vendors. (I don't know what percentage of that weight is actual)
Device manufacturers are popping up like mushrooms.